cluster-autoscaler/processors/nodegroupconfig/node_group_config_processor.go

```diff
@@ -33,6 +33,8 @@ type NodeGroupConfigProcessor interface {
 	GetScaleDownUtilizationThreshold(context *context.AutoscalingContext, nodeGroup cloudprovider.NodeGroup) (float64, error)
 	// GetScaleDownGpuUtilizationThreshold returns ScaleDownGpuUtilizationThreshold value that should be used for a given NodeGroup.
 	GetScaleDownGpuUtilizationThreshold(context *context.AutoscalingContext, nodeGroup cloudprovider.NodeGroup) (float64, error)
```

Comment:
Also this method should no long be necessary, sorry for missing that.

Reply:
Removed `GetScaleDownGpuUtilizationThreshold()`